forward_command_controller/include/forward_command_controller/forward_command_controller.h

```diff
@@ -85,14 +86,24 @@ class ForwardCommandController: public controller_interface::Controller<T>
   }
 
   void starting(const ros::Time& time);
-  void update(const ros::Time& time, const ros::Duration& period) {joint_.setCommand(command_);}
+  void update(const ros::Time& time, const ros::Duration& period)
+  {
+    tmp_command_ = *(command_.readFromRT());
+    joint_.setCommand(tmp_command_);
+    
+  }
 
   hardware_interface::JointHandle joint_;
-  double command_;
+  realtime_tools::RealtimeBuffer<double> command_;
+  double tmp_command_; // pre-allocated memory that is re-used to set the realtime buffer
```

Comment:
I thought about that, too, but then I decided to keep things similar, thus having the member variable for the ForwardCommandController as well as it can be found in the ForwardJointGroupCommandController and the effort_controllers::JointPositionController!

I was even wondering whether actually using a member variable may cause conflicts again between the callback and update function, i.e. whether we in general only should have local variables instead. Or is this resolved through the RealtimeBuffer?

@adolfo-rt: let me know how you would like to have it.